benchmarks/distributed/lambda_runtimes/cpp_runtime/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# Create CMake project
+cmake_minimum_required(VERSION 3.9)
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+#project(test-python LANGUAGES CXX)
+set(LAMBDA_NAME test-python)
+
+# Create AWS cpp Project
+find_package(aws-lambda-runtime)
```

Comment:
maybe add a check that this can be only compiled under appropriate Linux/docker. cf. https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Write-Platform-Checks